### DISCUSSION TITLE
Announce the selected GSoD technical writers

### DISCUSSION THREAD

**Discussion by alabulei1** [WasmEdge/docs#65] [Announcements]
Dear all, 

We are excited to announce the selected technical writers for the WasmEdge GSoD 2023 project. After careful consideration of all the applications we received, we have chosen two highly skilled and experienced writers @adithyaakrishna  and @mhmohona who we believe will excel in their roles. 

We would like to extend our sincere thanks to all who applied for this position. We were impressed by the level of talent and dedication displayed by each applicant.  It was a difficult decision as we received so many outstanding applications from talented and skilled writers. Unfortunately, we only have limited positions. 

Contributions for open-source projects are always welcome! Looking forward to working with you in the WasmEdge community.

Thank you again to everyone who applied.

Best regards,
The WasmEdge team

**Comment by mhmohona** [comment]
Thank you for selecting me. Excited to be part of the team and contribute to the WasmEdge community and look forward to working with you and using my skills and experience to produce high-quality technical documentation.

Thank you again for this opportunity, and I am excited to get started!

**Comment by kokamkarsahil** [comment]
Congrats @mhmohona and @adithyaakrishna !

**Reply by adithyaakrishna** [reply]
Thank you :)

**Comment by adithyaakrishna** [comment]
Thank you so much for selecting us! Excited to be a part of this and looking forward to working with y'all 🚀
